File: 2021/j1.java
// code by savir singh

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
public class Solve {
    public static void main(String[] args)
        throws IOException
    {
        BufferedReader b = new BufferedReader(
            new InputStreamReader(System.in));
        int bs = Integer.parseInt(b.readLine());
        int p = 5*bs-400;
        System.out.println(p);
        if (p==100) {
            System.out.println(0);
        }
        else if (p<100) {
            System.out.println(1);
        }
        else if (p>100) {
            System.out.println(-1);
        }
    }
}

File: 2021/j1.cpp
// code by savir singh

#include <iostream>
#include <string>
using namespace std;

int main(){

    int x;
    int y = 0;
    cin >> x;

    if(x==100) {
        y=1;
    }
    else if(x<100) {
        y=2;
    }
    else {
        ;
    }
    if (y==0) {
        int a = 5*x-400;
        cout << a;
        cout << "\n";
        cout << "-1";
    }
    else if(y==1) {
        int a = 5*x-400;
        cout << a;
        cout << "\n";
        cout << "0";
    }
    else {
        int a = 5*x-400;
        cout << a;
        cout << "\n";
        cout << "1";
    }
    return 0;
}
